### PR TITLE
Fix unit tests from breaking

### PR DESCRIPTION
**Pull request by piraka9011**

### CODE REVIEW

File: cross_compile/sysroot_compiler.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 
 CC_BUILD_SETUP_FILE_TEMPLATE = Template(
-    'shell=`echo $$SHELL | awk -F/ \'{print $$NF}\'`\n'
+    'shell=`echo $$SHELL | awk -F/ \'{print $$NF}\'`\n'  # noqa
```
Comment (thomas-moulard) [inline cross_compile/sysroot_compiler.py]: noqa is a last-resort mechanism - what's the issue?

**Review by thomas-moulard — APPROVED** [review]

File: cross_compile/sysroot_compiler.py [hunk 2]
```diff
@@ -248,48 +272,56 @@ def build_workspace_sysroot_image(self) -> None:
...
             # We also want to remove newline (\n) and carriage returns (\r) to
             # avoid mangled output.
             error_line = chunk.get('error', None)
             if error_line:
-                logger.exception('Error building sysroot image. The following error'
-                                 ' was caught:\n{}'.format(error_line))
-                raise docker.errors.BuildError(reason=error_line, build_log=error_line)
+                logger.exception(
+                    'Error building sysroot image. The following error'
+                    ' was caught:\n{}'.format(error_line))
```
Comment (piraka9011) [inline cross_compile/sysroot_compiler.py:282]: Fix format

File: cross_compile/sysroot_compiler.py [hunk 3]
```diff
@@ -248,48 +272,56 @@ def build_workspace_sysroot_image(self) -> None:
...
-        logger.info('Exporting sysroot to path [{}]'.format(self._target_sysroot))
+        logger.info('Exporting sysroot to path [%s]', self._target_sysroot)
         shutil.rmtree(str(self._target_sysroot), ignore_errors=True)
         # TODO: Use context to make sure temp directory doesn't leak
         tmp_sysroot_dir = tempfile.mkdtemp(suffix='-cc_build')
-        sysroot_tarball_path = Path(tmp_sysroot_dir) / (SYSROOT_DIR_NAME + '.tar')
+        sysroot_tarball_path = (
+            Path(tmp_sysroot_dir) / (SYSROOT_DIR_NAME + '.tar'))
         image_tag = self._platform.get_workspace_image_tag()
         logger.info('Exporting filesystem of image {} into tarball {}'.format(
```
Comment (piraka9011) [inline cross_compile/sysroot_compiler.py:301]: format

File: test/test_sysroot_compiler.py [hunk 4]
```diff
@@ -83,19 +83,25 @@ def test_get_workspace_image_tag(platform_config):
 def test_docker_config_args(docker_config):
     """Make sure the Docker configuration is setup correctly."""
     args = _default_args()
-    test_config_string = 'Base Image: {}\nNetwork Mode: {}\nCaching: {}'.format(
-        args.sysroot_base_image, args.docker_network_mode, args.sysroot_nocache)
+    test_config_string = (
+        'Base Image: {}\n'
+        'Network Mode: {}\n'
+        'Caching: {}').format(
+        args.sysroot_base_image, args.docker_network_mode,
```
Comment (piraka9011) [inline test/test_sysroot_compiler.py:90]: format

**Comment by thomas-moulard** [comment]
Merging this as it reduces the number of failures on this package already. Tackling e2e failures will be handled in a separate PR.

**Pull request merged**